src/proc/color-formats-converter.cpp

```diff
@@ -46,6 +46,294 @@ bool has_avx()
 
 namespace librealsense 
 {
+    void convert_yuv_to_rgb( const byte yuv[3], byte * rgb )
```

Comment:
This is the same as in the unit-test?
Reusable?

Reply:
will remove the unit tests for now